typings/index.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference path="globals/selenium-webdriver/index.d.ts" />
```

Comment:
Can we add a note that these are forked from definitely typed so we can upkeep them?